packages/web/src/components/spa/HackAUBG/JourneySection/JourneyMobile/JourneyMobile.jsx

```diff
@@ -12,6 +12,7 @@ export const JourneyMobile = () => {
             </div>
             <div className="journey-mobile-content">
                 <JourneyStep
+                    id="1"
                     title={
                         <h2>
```

Comment:
On another note, why the tags are placed here? Shouldn't you pass only the string without the elements (<br>, <s>, <em> tags are exceptions to this) ? Please extract the header tags inside the JourneyStep component

Reply:
so i put the id tags there cuz i want ti get the length of the component and before when one is opened it affected all other components

Reply:
i will look at it